Weather/Weather/Data/Sources/APIService.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ServiceManager.swift
+//  Weather
+//
+//  Created by minh duc on 2/15/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import Then
+
+struct BaseURL {
+    static let place = "https://maps.googleapis.com/maps/api/place/autocomplete/json?"
+}
+
+enum ResponseManager {
+    case success(data: [String: Any])
+    case failed(error: LocalError)
+}
+
+enum LocalError: Error {
+    case InvalidURL
+    case InvalidAcceptType
+    case NotFound
+    case Success
+    case Unexpected(message: Error?)
+}
+
+class APIService {
+    private let googleKey = "AIzaSyBhYoKGI05fGWSx2F6uz1Wa1wt87v2nDlE"
+    
+    private let session = URLSession(configuration: .default).then {
+        $0.configuration.timeoutIntervalForRequest = 30
+        $0.configuration.timeoutIntervalForResource = 30
+    }
+    
+    func searchAutoComplete(with place: String, completion: @escaping(ResponseManager)->Void ) {
```

Comment:
sai coding convention
```suggestion
    func searchAutoComplete(with place: String, completion: @escaping(ResponseManager) -> Void) {
```